interfaces/tests/file_int/file_int_helpers_win32.c

```diff
@@ -0,0 +1,34 @@
+//Copyright(c) Microsoft.All rights reserved.
+//Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+#ifdef __cplusplus
+#include <cstdlib>
+#include <cstdio>
+#include <cstring>
+#else
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#endif
+#include "file_int_helpers.h"
+
+void delete_file(const char* filename)
+{
+    char* command = malloc(strlen(filename) + 5);

```

Comment:
> malloc(strlen(filename) + 5) [](start = 19, length = 29)

malloc can fail, check this for NULL. Then this function probably needs to return an int (but you could just ignore the return in the test code `(void)delete_file(...)` because you will catch important issues with `check_file_exists`.